src/Microsoft.PowerShell.Commands.Utility/commands/utility/CsvCommands.cs

```diff
@@ -244,14 +246,6 @@ protected override void BeginProcessing()
         {
             base.BeginProcessing();
 
-            // Validate that they don't provide both Path and LiteralPath, but have provided at least one.
-            if (!(_specifiedPath ^ _isLiteralPath))
-            {
-                InvalidOperationException exception = new InvalidOperationException(CsvCommandStrings.CannotSpecifyPathAndLiteralPath);
```

Comment:
You should remove the resource string from resx file too.

Reply:
@Maamue if you have trouble finding that, do a search through the repo for the string name (`CannotSpecifyPathAndLiteralPath`) and you're looking for the resx file as Ilya mentioned, not the C# one (which is generated from the resx file at build time). 🙂

Reply:
Thank you for the hints! I found it and removed it. Tests are in progress :)